docs/guides/serverless-manage-resources.ipynb

```diff
@@ -133,18 +340,27 @@
    ]
   },
   {
-   "cell_type": "markdown",
+   "cell_type": "code",
+   "execution_count": 10,
    "id": "b9b38d02-1c4b-4598-abc8-f2ff2f3c1d23",
    "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Writing ./source_files/transpile_parallel.py\n"
+     ]
+    }
+   ],
    "source": [
-    "```python\n",
-    "# source_files/transpile_parallel.py\n",
+    "%%writefile ./source_files/transpile_parallel.py\n",
```

Comment:
Do you know why we change file names here?

Reply:
Not really, I guess they could both be `transpile_remote`